docs/reference/connector/apis/update-connector-filtering-api.asciidoc

```diff
@@ -184,3 +140,64 @@ PUT _connector/my-connector/_filtering
     "result": "updated"
 }
 ----
+
+The following example updates the draft {enterprise-search-ref}/sync-rules.html#sync-rules-advanced[advanced sync rules] for the MySQL connector with id `my-sql-connector`.  Advanced sync rules are specific to each connector type. For more details on the syntax and rules specific to the MySQL connector in our example, you can refer to the {enterprise-search-ref}/connectors-mysql.html#connectors-mysql-sync-rules[MySQL connector documentation] .
```

Comment:
```suggestion
The following example updates the draft advanced sync rules for a MySQL connector with id `my-sql-connector`. Advanced sync rules are specific to each connector type. Refer to the references for connectors that support {enterprise-search-ref}/sync-rules.html#sync-rules-advancedl[advanced sync rules] for syntax and examples.
```

Reply:
```{enterprise-search-ref}/connectors-mysql.html#connectors-mysql-sync-rules[MySQL connector documentation] .```

Ahh I commited this but now I see that you changed this to a generic documentation of advanced sync rules - since they are connector-specific I wanted to link them to mysql definition of advances sync rule.

Reply:
Yeah that was an opinionated choice to just say "this is an example, not all connectors support advanced sync rules, here's the top-level docs page that lists connectors that support advanced rules"

Feel free to change around again if you think it makes sense to have specific link :)